samples/dashboards/components/highcharts-column-assignment-2d-data/demo.html

```diff
@@ -0,0 +1,7 @@
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/datagrid/datagrid.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/layout.js"></script>
+
+<div id="container"></div>
```

Comment:
I would choose a slightly smaller data set because we can't see the points from the table right now thanks to data grouping.